tests/cypress/e2e/features/consensus.js

```diff
@@ -8,34 +8,48 @@ import { translatePoints } from '../../support/utils';
 
 context('Basic manipulations with consensus job replicas', () => {
     const labelName = 'Consensus';
-    const taskName = 'Test consensus';
     const serverFiles = ['archive.zip'];
     const replicas = 4;
-    const taskSpec = {
-        name: taskName,
+    const dataSpec = {
+        server_files: serverFiles,
+        image_quality: 70,
+    };
+    const extras = { consensus_replicas: replicas };
+
+    const taskNameCreate = 'Test consensus creation';
```

Comment:
I would suggest to use 1 task spec. In before of each `describe` use `headlessCreateTask` and delete it in `after`